src/patternfly/utilities/Text/examples/Text.md

```diff
@@ -0,0 +1,128 @@
+---
+id: Text
+section: utilities
+---
+
+import './Text.css'
+
+## Examples
+
+### Font size
+
+```hbs
+{{#> text text--modifier="pf-u-font-size-xs"}}
+  Extra small text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-sm"}}
+  Small text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-md"}}
+  Medium text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-lg"}}
+  Large text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-xl"}}
+  Extra large text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-2xl"}}
+  2xl text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-3xl"}}
+  3xl text
+{{/text}}
+{{#> text text--modifier="pf-u-font-size-4xl"}}
+  4xl text
+{{/text}}
+```
+
+### Font weight
+
+```hbs
+{{#> text text--modifier="pf-u-font-weight-light"}}
+  Light
+{{/text}}
+{{#> text text--modifier="pf-u-font-weight-normal"}}
+  Normal
+{{/text}}
+{{#> text text--modifier="pf-u-font-weight-bold"}}
+  Bold
+{{/text}}
+```
+
+### Color
+
+```hbs
+{{#> text text--modifier="pf-u-color-100"}}
+  Font color 100
+{{/text}}
+{{#> text text--modifier="pf-u-color-200"}}
+  Font color 200
+{{/text}}
+{{#> text text--modifier="pf-u-color-300"}}
+  Font color 300
+{{/text}}
+{{#> text text--modifier="pf-u-color-400"}}
+  Font color 400
+{{/text}}
+```
+
+### Controlling text
+
+```hbs
+<h4><strong>Breakword</strong></h4>
+{{#> text text--modifier="pf-u-font-break-word"}}
```

Comment:
Need to update the utility name of the "controlling text" utilitys to `pf-u-text` like you have in the docs.

```suggestion
{{#> text text--modifier="pf-u-text-break-word"}}
```